pkg/providers/vsphere/setupuser/setupuser.go

```diff
@@ -0,0 +1,187 @@
+package setupuser
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/aws/eks-anywhere/pkg/config"
+	"github.com/aws/eks-anywhere/pkg/logger"
+	"github.com/aws/eks-anywhere/pkg/providers/vsphere"
+)
+
+const (
+	vSphereRootPath = "/"
+)
+
+func SetupGOVCEnv(ctx context.Context, vsuc *VSphereSetupUserConfig) error {
+	os.Setenv("GOVC_URL", vsuc.Spec.Connection.Server)
+	os.Setenv("GOVC_INSECURE", strconv.FormatBool(vsuc.Spec.Connection.Insecure))
+	os.Setenv("GOVC_DATACENTER", vsuc.Spec.Datacenter)
+	return nil
+}
+
+func Run(ctx context.Context, vsuc *VSphereSetupUserConfig, govc vsphere.ProviderGovcClient) error {
+	err := createGroup(ctx, vsuc, govc)
+	if err != nil {
+		return err
+	}
+
+	err = addUserToGroup(ctx, vsuc, govc)
+	if err != nil {
+		return err
+	}
+
+	err = createRoles(ctx, vsuc, govc)
+	if err != nil {
+		return err
+	}
+
+	err = associateRolesToObjects(ctx, vsuc, govc)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func CreateUser(ctx context.Context, vsuc *VSphereSetupUserConfig, govc vsphere.ProviderGovcClient, password string) error {
```

Comment:
This function seems redundant. We don't handle the error at all so it can be abbreviated to:

```go
func CreateUser(ctx context.Context, vsuc *VSphereSetupUserConfig, govc vsphere.ProviderGovcClient, password string) error {
	return govc.CreateUser(ctx, ...)
}
```

The caller can just call `govc` directly.